feature/mtu/largeippacket/otg_tests/large_ip_packet_transmission/README.md

```diff
@@ -7,31 +7,90 @@ IPv4 and IPv6 packet sizes are sent over them.
 
 ## Procedure
 
-* Configure DUT with routed input and output interfaces with an Ethernet MTU of 9216.
+* Test environment setup
+  * Configure DUT with routed input and output interfaces with an Ethernet MTU of 9216.
   * Test should be executed with two different interface/connectivity profiles:
     1) Standalone -- one input and one output port
     2) Bundle with four input members and four output members
-* Run traffic flows of the following size over IPv4 and IPv6 between ATE ports. 
-  * 1500 Bytes
-  * 2000 Bytes
-  * 4000 Bytes
-  * 9202 Bytes
-* Assert ATE reports packets sent and received count are the same, indicating no fragmentation, and 
-  successful transit.
+
+* MTU-1.3.1: Test with Physical and Bundle interfaces
+  * Run traffic flows of the following size over IPv4 and IPv6 between ATE ports. 
+    * 1500 Bytes
+    * 2000 Bytes
+    * 4000 Bytes
+    * 9202 Bytes
+  * Assert ATE reports packets sent and received count are the same, indicating no fragmentation, and 
+    successful transit.
+
+* MTU-1.3.2: Test w/ tunnel interfaces and forwarding plane via the physical and bundled interfaces
+  * Configure DUT with a GRE tunnel interface.
```

Comment:
Something we should clarify by providing the canonical OC is:

From the paths included at the end of the readme, I am guessing the intent is for the stateful interface style configuration.  This should be the target if one is going to run a control protocol over it (such as bgp)?  

The other style would be a policy-forwarding configuration which uses match and encap rules.   Or perhaps you have use cases for both styles? (which should have two readmes if that is true)

The test should define 'canonical OC' used for the test intent.  The follow format is [pending standardization for README](https://github.com/openconfig/featureprofiles/pull/3336).  I suggest using it here:

### OC tunnel interface configuration
```yaml
interfaces:
  - interface: 'tunnel1_if_name'
    config:
      name: 'tunnel1_if_name'
    tunnel:     # configures the tunnel outer packet 
      config:
       src: 'tunnel1_outer_ip_src'
       dst: 'tunnel1_outer_ip_dst'
       ttl: 'tunnel1_outer_ttl'
       gre-key: 'tunnel1_outer_gre_key'
      ipv4:
        config:
          mtu: 'tunnel1_inner_mtu'
        addresses:
          - address:
            config:
              ip: 'tunnel1_inner_ipv4'
              prefix-length: 'tunnel1_inner_ipv4_prefixlen'
      ipv6:
        config:
          mtu: 'tunnel1_inner_mtu'        addresses:
        addresses:
          - address:
            config:
              ip: 'tunnel1_inner_ipv6'
              prefix-length: 'tunnel1_inner_ipv6_prefixlen'
```

Reply:
1. Do we have a use case for both scenarios?
**Response:**

Just to elaborate on the topics in the question so we set the context for the requirements.

a. Filter based encap: This approach uses a filter on the incoming direction that identifies, L3/L4 or other parameters in the TOS byte to set a forwarding action as encap to tunnel destination address which is reachable via the FIB entry

b. Interface based encap: In this case the router has virtual tunnel interfaces. The DUT will have routes for the destination address of the received traffic pointing at the tunnel interface. The tunnel interface has required encap parameters and the tunnel destination configured. The tunnel destination will be reachable via LPM lookup

While "a" provides us mature filter options to tunnel the desired flow, "b" provides a simple L3 matched tunelling that is independent of ACLs. This is important because, large filters consume memory resources and we should avoid them where possible.

So the answer is, yes we need both cases. We currently have Tun-1 and Tun-2  tests that cover both scenarios. What is missing is something that covers a hybrid of both. We will definitely add the latter use case.

Reply:
Ok, I think you are saying we use both and why.  Thanks!

I am not sure we need to have both together in one test, unless it is a special scaling test.  In my experience, unless you're looking at interop or performance of a very small number of features,  a config scale/IC resources limit test is best added to an integration test (with a full device config push) and probably doesn't fit into FNT.  (because config scale requires configuring a lot of features at the same time).